README.md

```diff
@@ -38,16 +36,14 @@ Make sure you have installed all of the following prerequisites on your developm
   ```bash
   git clone --depth=1 https://github.com/commercetools/nodejs-boilerplate.git
   ```
-  2. Clean up and install
+  2. Clean up, setup and install.  
+  Takes the project name and an optional second directory like `bin` to use.
```

Comment:
Ah ok, so the adding to the watch task is what it really gives you. Maybe we can explain this a bit better, by telling that the additional directory will be automatically added to the watch task.